src/main/java/reposense/parser/ArgsParser.java

```diff
@@ -92,6 +93,11 @@ private static ArgumentParser getArgumentParser() {
                         + "If not provided, default file formats will be used.\n"
                         + "Please refer to userguide for more information.");
 
+        parser.addArgument("-ignorejson")
+                .setDefault(false)
```

Comment:
Hmm, tested this, and it seems having the `.action(Arguments.storeTrue())` will automatically set this argument to be default `false`, so this is probably not needed here.